blog/2024/10-27-media-segments-10.10.0/index.mdx

```diff
@@ -0,0 +1,85 @@
+Jellyfin only stores and provides this data to clients but to generate it you need a plugin that provide this data to jellyfin to be stored.
+
+We did create a plugin to create segments based on Chapter names which sounds similar but is something different. 
+A Chapter is just any section of a video, but without any defined type. 
+A MediaSegment on the other hand is a section in a video, with a defined type.
+
+## Plugin support
+
+Media Segment plugins can now utilise our new system to store their information about Intros/Outros/Commericals and all other types of segments. 
+This information is then used by clients via our Api to provide actions like a "Skip" button in their UI.
+The big benefit here is that this approach generalises how Segments are handled, so gone are the days where you needed to inject javascript into the webUI to get IntroSkipping.
+
+This does *NOT* make the existing plugins obsolete, however as we still rely on them to generate the segments in the first place. 
+But as more clients implement support, we can bring segments to more clients and provide for example IntroSkipping.
+
+## Clients
+
+Media segments are currently supported in the web interface and the Android TV[^1] app.
+
+[^1]: Available starting from 0.18, currently [in beta](/posts/android-betas)
+The server however just delivers the information about where the segment is in the video and of what type it is, it remains up the the individual client to decide what to do with that information.
+
+As the whole system of Media Segments is quite new, please do not expect many clients to support it.
+
+*The ATV client did not yet release a version with MediaSegment support.
+
+## Step by step
+
+If you want to utilise Media Segments you have to do the following:
+
+1. Update your jellyfin server to 10.10.
+2. Install a plugin that creates MediaSegments via the plugin catalog. For example our official "Chapter Segments Provider".
+3. Restart the jellyfin server to enable the plugin.
+4. Run the "Scan Mediasegments" scheduled task and wait for it to finish.
+
+This will populate the MediaSegments for use by a client. 
+For the WebUI you also need to set what should happen when there is a MediaSegment. 
+This needs to be set via each individual users "Profile -> Playback -> Media Segment Actions".
+
+## The future
+
+This is just the first step for MediaSegments. 
+The underlying architecture allows for much more which we are now able to explore. 
+We also allow for multiple plugins to provide segments, so you can pick any one plugin or all of them to provide the best experience.
+
+## Addendum
+
+Hi i am JPVenson and i would like to introduce myself as a new Server-Team member for the jellyfin FOSS project.
+
+I have been active for a bit now, first just as an Freelance software developer that created its own small media server because i did not wanted to pay for other serivces but then quickly learned what it all takes.
```

Comment:
```suggestion
I have been active for a bit now, first just as an Freelance software developer that created its own small media server because I did not wanted to pay for other services but then quickly learned what it all takes.
```